airflow/providers/apache/beam/provider.yaml

```diff
@@ -41,6 +41,3 @@ hooks:
   - integration-name: Apache Beam
     python-modules:
       - airflow.providers.apache.beam.hooks.beam
-
-hook-class-names:
-  - airflow.providers.apache.beam.hooks.beam.BeamHook
```

Comment:
Is it not better to add `conn_type`, `conn_name_attr` and `hook_name` to `BeamHook` instead of deleting these lines?
(e.g in Casssanda: https://github.com/apache/airflow/blob/b34f236eb657553daac855c4b833db37fc3e56ce/airflow/providers/apache/cassandra/hooks/cassandra.py#L86)

Reply:
I think it's a bit tricky for `BeamHook` to have those attributes because Beam doesn't tie to a specified "backend", it can be run on Dataflow, Flink, Spark, etc. and so their credentials may also come in different forms. I think it's possible, but it may require some extra thoughts on what the `Connection` object for Beam will look like. For now, we can at least get rid of those warnings whenever `ProvidersManager` is instantiated.

Reply:
Agree. It is better just to remove it for now.